src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -1362,6 +1371,11 @@ public bool IsUsingOrExternOrImport(SyntaxNode node)
                    node.IsKind(SyntaxKind.ExternAliasDirective);
         }
 
+        public bool IsUsingOrImport(SyntaxNode node)
+        {
+            return node.IsKind(SyntaxKind.UsingDirective);
+        }
```

Comment:
we already have: IsUsingOrExternOrImport

that's likely sufficient for your usecase.

Reply:
You are right, reverted